### DISCUSSION TITLE
Is there way for WikiTraccs to migrate calendar and comments?

### DISCUSSION THREAD

**Discussion by novice002** [WikiTransformationProject/wikitraccs-releases#122] [Q&A] [answered]
Hello,

Is there way to migrate the following from Confluence? 

- Calendar 
- Comments (added at the bottom of a page - I understand attachment/file added via comment is still migrated)

I have the feeling the answer would be 'no', but want to make sure I don't miss any opportunity if any.

Thank you!

**Comment by heinrich-ulbricht** [comment]
Calendars are not explicitly handled, so they would be lost. Never thought about them, to be honest. If there is a place in SharePoint where you would like them to go, please tell me :)

Footer comments can be migrated, they'll become page content in SharePoint, for lack of a better place to put them.

**Reply by novice002** [reply]
Thank you for the information, Heinrich

I understand for the Calendar part - as there is no intrinsic calendar feature in the modern sites in SPOnline 
(We disabled classic site template).

As for the footer Comments, I still don't see them migrated so far unfortunately :-(
Can you think of any permission requirement on Space, which may cause the comment truncation?

The migration account is given the following permissions on our Confluence (ver - 7.0.x):
- confluence-administrator 
- Confluence System Administrator
- Space Admin permission (explicitly given on Space level)

Thank you!

**Reply by heinrich-ulbricht** [reply, marked as answer]
Hm, did you enable comment migration in the settings?
![image](https://github.com/WikiTransformationProject/wikitraccs-releases/assets/3469970/c7e19e00-a857-460c-b0dc-942011f9472b)

**Reply by novice002** [reply]
Ouch... that was it! 
I am now seeing all the footer comments migrated (never imagined these comments would look 'beautiful' to me)

THIS CERTAINLY IS USER ERROR.

Thank so much for your guidance.
Much appreciated!

**Reply by heinrich-ulbricht** [reply]
No worries. This is a WikiTraccs issue as well as it could more clearly communicate and validate with the user what it will migrate. Like a summary dialog when starting the migration. Have been thinking about that for a while.

**Reply by jlriverayuso** [reply]
Could Wikitraccs, convert the calendar to a SharePoint List? I understand that the recurrence no longer works in SharePoint Online but is it an option?

**Reply by heinrich-ulbricht** [reply]
@jlriverayuso Currently WikiTraccs can't.

I'd have to check if it's technically possible. Sounds like it should be possible. Could you show me a sample of how you would expect the result to look? Take a Confluence calendar, create a SharePoint list with some elements in it, post 2 screenshots for me to look at? Just to make sure I understand. I'd consider this for the roadmap.

**Reply by jlriverayuso** [reply]
@heinrich-ulbricht 

Here is the Calendar list in SharePoint:
![image](https://github.com/user-attachments/assets/e5b07b31-2924-4934-9273-d18faee2b576)

Same Calendar in Confluence:
![image](https://github.com/user-attachments/assets/c55f53ab-1876-4253-a16b-7b073427e466)

**Comment by heinrich-ulbricht** [comment]
Update: Calendar migration is in the making and will be included in one of the upcoming WikiTraccs releases.
